<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_func</class>
 <widget class="QWidget" name="main_func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>135</width>
     <height>216</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="ip_label">
      <property name="text">
       <string>我的ip地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip_line"/>
    </item>
    <item>
     <widget class="QLabel" name="he_ip_label">
      <property name="text">
       <string>对面的ip地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="he_ip_line"/>
    </item>
    <item>
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>我的名字</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_line"/>
    </item>
    <item>
     <widget class="QLabel" name="port_label">
      <property name="text">
       <string>自己端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port_line"/>
    </item>
    <item>
     <widget class="QLabel" name="he_port_label">
      <property name="text">
       <string>对方端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="he_port_line"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="drawButton">
      <property name="text">
       <string>文件接收</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="file_send_Button">
      <property name="text">
       <string>文件发送</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="file_shared_Button">
      <property name="text">
       <string>文档共写</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="talkvideoButton">
      <property name="text">
       <string>聊天</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="set_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
